lib/semian/pid_controller.rb

```diff
@@ -10,11 +10,7 @@ module Semian
   # Note: P increases when error_rate increases
   #       P decreases when rejection_rate increases (providing feedback)
   class PIDController
-    attr_reader :name, :rejection_rate
-
-    def initialize(name:, kp: 1.0, ki: 0.1, kd: 0.0, target_error_rate: nil,
```

Comment:
removed:
* Name: not useful
* target_error_rate: not useful
* Default values: they are getting set in semian.rb